### ISSUE TITLE
Bypassing D-Bus activation means xdg-desktop-portal-cosmic is racing with xdg-desktop-portal

### ISSUE THREAD

**Issue by alyssais** [pop-os/xdg-desktop-portal-cosmic#117]
When xdg-desktop-portal starts, it will try to create proxies for the portals offered by xdg-desktop-portal-cosmic.  If those portals don't already exist, it will try to activate xdg-desktop-portal-cosmic through D-Bus.  xdg-desktop-portal-cosmic's D-Bus file just runs false, so xdg-desktop-portal gives up on trying to run that portal — it doesn't expect that a service that fails to start will somehow start itself later on.  This means that there will be no COSMIC portals for the whole session, unless xdg-desktop-portal is restarted.

```
Failed to create screencost proxy: Error calling StartServiceByName for org.freedesktop.impl.portal.desktop.cosmic: Process org.freedesktop.impl.portal.desktop.cosmic exited with status 1
````

For me, this meant that updating from alpha 2 to alpha 3 broke my COSMIC portals, because libcosmic's font preloading was enough to make starting xdg-desktop-portal-cosmic consistently slow enough not to register its name in time for xdg-desktop-portal trying to create its proxies.

**Comment by Drakulix** [comment]
yeah we need to fix our approach to privileged protocols and change cosmic-session to not launch xdg-desktop-portal anymore..
